src/app/I18n/FR.ts

```diff
@@ -370,12 +370,15 @@ export const FR = {
         placeholderSubtitle: 'Les équipes s’activent pour fournir les premiers résultats !',
         columns: {
           team: 'Équipe',
-          answeredQuestions: 'Questions répondues',
+          answeredQuestions: 'Taux de réponses',
```

Comment:
maybe you should change the label of this property for something like "answerRate" @Hugonotfound, no  ?